src/DomainLib.EventStore.AspNet/ServiceCollectionExtensions.cs

```diff
@@ -15,17 +14,17 @@ public static class ServiceCollectionExtensions
         /// <summary>
         /// Adds an aggregate repository that uses EventStoreDb to persist events 
         /// </summary>
-        public static void AddEventStoreAggregateRepository(this IServiceCollection services,
-                                                            IConfiguration configuration,
-                                                            AggregateRegistry<object, object> aggregateRegistry)
+        public static IServiceCollection AddEventStoreAggregateRepository(this IServiceCollection services,
+                                                                          IConfiguration configuration,
+                                                                          AggregateRegistry<object, object> aggregateRegistry)
         {
-            AddEventStoreAggregateRepository<object, object>(services, configuration, aggregateRegistry);
+            return AddEventStoreAggregateRepository<object, object>(services, configuration, aggregateRegistry);
         }
 
         /// <summary>
         /// Adds an aggregate repository that uses EventStoreDb to persist events 
         /// </summary>
-        public static void AddEventStoreAggregateRepository<TCommandBase, TEventBase>(this IServiceCollection services,
+        public static IServiceCollection AddEventStoreAggregateRepository<TCommandBase, TEventBase>(this IServiceCollection services,
```

Comment:
Why the change of signature?

Most service collection extension methods do it like how you had it before 😛

Reply:
I've seen quite a few that return the service collection and allow you to chain them.
Figured I'd go for that pattern